### PR TITLE
GG-23552 Add DR clearStore task to Web Console

### PR DESCRIPTION
**Pull request by nva**

### CODE REVIEW

File: modules/core/src/main/java/org/apache/ignite/internal/IgniteFeatures.java [hunk 1]
```diff
@@ -115,6 +115,9 @@
     /** */
     MANAGEMENT_CONSOLE(28),
 
+    /** Cluster has task to clear sender store. */
+    WC_DR_CLEAR_SENDER_STORE(29),
+
```
Comment (vsisko) [inline modules/core/src/main/java/org/apache/ignite/internal/IgniteFeatures.java]: We should check that gridgain-core library in classpath.

Comment (nva) [reply modules/core/src/main/java/org/apache/ignite/internal/IgniteFeatures.java]: Why and how we can check this in ignite-core module?
This ticket contains changes in ggprivate repository with tasks

Comment (akuznetsov-gridgain) [reply modules/core/src/main/java/org/apache/ignite/internal/IgniteFeatures.java]: We should check DR plugin. I think it can be done via classForName() ?

Comment (nva) [reply modules/core/src/main/java/org/apache/ignite/internal/IgniteFeatures.java]: We can't add checking of gridgain-core class in gridgain-ce code. In same time we has additional checks for that in our frontend/web-console code

**Pull request merged**